module-os/board/linux/fsl_runtimestat_gpt.c

```diff
@@ -1,59 +0,0 @@
-// Copyright (c) 2017-2020, Mudita Sp. z.o.o. All rights reserved.
```

Comment:
where is it now?

Reply:
It is no longer required as the new linux port implements it in `portmacro.h` and `port.c`.